x-pack/test/idp-fixture/src/main/java/org/elasticsearch/test/fixtures/idp/IdpTestContainer.java

```diff
@@ -35,107 +33,7 @@ protected IdpTestContainer() {
-                            + " && echo \"$logback_classic_hash  logback-classic-$logback_version.jar\" | sha1sum -c -"
-                            + " && mv logback-classic-$logback_version.jar /opt/shib-jetty-base/lib/logging/"
-                    )
-                    // Download the logback-core library for Jetty logging, verify the hash, and place
-                    .run(
-                        "wget -q https://repo.maven.apache.org/maven2/ch/qos/logback/logback-core/$logback_version/logback-core-$logback_version.jar"
-                            + " && echo \"$logback_core_hash  logback-core-$logback_version.jar\" | sha1sum -c -"
-                            + " && mv logback-core-$logback_version.jar /opt/shib-jetty-base/lib/logging/"
-                    )
-                    // Download the logback-access library for Jetty logging, verify the hash, and place
-                    .run(
-                        "wget -q https://repo.maven.apache.org/maven2/ch/qos/logback/logback-access/$logback_version/logback-access-$logback_version.jar"
-                            + " && echo \"$logback_access_hash  logback-access-$logback_version.jar\" | sha1sum -c -"
-                            + " && mv logback-access-$logback_version.jar /opt/shib-jetty-base/lib/logging/"
-                    )
-                    // ## Copy local files
-                    .copy("idp/shib-jetty-base/", "/opt/shib-jetty-base/")
-                    .copy("idp/shibboleth-idp/", "/opt/shibboleth-idp/")
-                    .copy("idp/bin/", "/usr/local/bin/")
-                    // Setting owner ownership and permissions
-                    .run(
-                        "useradd jetty -U -s /bin/false"
-                            + " && chown -R root:jetty /opt"
-                            + " && chmod -R 640 /opt"
-                            + " && chown -R root:jetty /opt/shib-jetty-base"
-                            + " && chmod -R 640 /opt/shib-jetty-base"
-                            + " && chmod -R 750 /opt/shibboleth-idp/bin"
-                    )
-                    .run("chmod 750 /usr/local/bin/run-jetty.sh /usr/local/bin/init-idp.sh")
-                    .run("chmod +x /opt/jetty-home/bin/jetty.sh")
-                    // Opening 4443 (browser TLS), 8443 (mutual auth TLS)
-                    .cmd("run-jetty.sh")
-                    .withStatement(
-                        new SingleArgumentStatement(
-                            "HEALTHCHECK",
-                            "CMD curl -f -s --http0.9 http://localhost:4443 " + "--connect-timeout 10 --max-time 10 --output - > /dev/null"
-                        )
-                    )
-                    // .expose(4443)
-                    .build()
-            )
+            new ImageFromDockerfile("es-idp-testfixture").withDockerfileFromBuilder(builder -> builder.from(DOCKER_BASE_IMAGE).build())
```

Comment:
I think we want to use `RemoteDockerImage` here instead. That should avoid building essentially an "empty" Dockerfile that just extends from the remote one.